azure-template/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://dev.azure.com/azure-sdk/public/_apis/build/status/azure-sdk-for-python.client?branchName=master
+    :target: https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46?branchName=master
+
+Azure Template Package
+======================
+
+This template package matches necessary patterns that the development team has established to create a unified sdk functional from Python 2.7 onwards. The packages contained herein can be installed singly or as part of the `azure` namespace. Any other introductory text should go here.
+
+This package has been tested with Python 2.7, 3.4, 3.5, 3.6 and 3.7.
+
+For a more complete set of Azure libraries, see the `azure <https://pypi.python.org/pypi/azure>`__ bundle package.
+
+
+Compatibility
```

Comment:
I think it's enough, and we should stop having this message.  It's been 3.5 years, I don't exect at this point upset customer because they want to update from azure 0.11.
I shall remove that as well from my own template.

Reply:
Ok. So I should remove the Compatibility section entirely?